src/object-store/src/cache_policy.rs

```diff
@@ -89,12 +92,16 @@ impl<I: Accessor, C: Accessor> LayeredAccessor for LruCacheAccessor<I, C> {
 
         match self.cache.read(&cache_path, OpRead::default()).await {
             Ok((rp, r)) => {
+                increment_counter!(OBJECT_STORE_LRU_CACHE_HIT);
```

Comment:
According to our code style, prefer to use `metrics::OBJECT_STORE_LRU_CACHE_HIT` instead.

Reply:
Thanks. I'm willing to make the change, but I'm not sure what is the best way. The current version is copied from [`runtime.rs`](https://github.com/GreptimeTeam/greptimedb/blob/f4190cfca6f2c8b8cfb4a0b94b95df357c08b2b5/src/common/runtime/src/runtime.rs#L156). Simplying qualifying it doesn't seem to work, since it conflicts with the external `metrics` crate. Also, qualifing the label in the macro would cause unexpected token errors. I also tried aliasing the external crate like `extern crate metrics as metrics_lib`, but the build failed.

Let me know if you have better ideas. Thanks a lot.

Reply:
It will not conflict with the external `metrics` crate unless you `use metrics` itself.

Sorry, the `runtime.rs` is misleading, I'll correct it.

Reply:
Sorry, I'm not very familiar with Rust. Maybe I miss something. Just changing to `metrics::OBJECT_STORE_LRU_CACHE_HIT` would break the build (IIUC, compiler is confused which`metrics` should be used). Using `crate::metrics::OBJECT_STORE_LRU_CACHE_HIT` can pass the build, but I'm not sure if it meets the code style.

Reply:
You can `use crate::metrics`, then you can use `metrics::OBJECT_STORE_LRU_CACHE_HIT` in the current mod.

Reply:
That's what I tried. The compiler would fail to resolve `use metrics::increment_counter;`. 

```
error[E0659]: `metrics` is ambiguous
  --> src/object-store/src/cache_policy.rs:22:5
   |
22 | use metrics::increment_counter;
   |     ^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `metrics` could refer to a crate passed with `--extern`
   = help: use `::metrics` to refer to this crate unambiguously
note: `metrics` could also refer to the module imported here
  --> src/object-store/src/cache_policy.rs:29:5
   |
29 | use crate::metrics;
   |     ^^^^^^^^^^^^^^
   = help: use `self::metrics` to refer to this module unambiguously
```